allauthdemo/management/commands/setup_project.py

```diff
@@ -34,7 +34,7 @@ def create_admin(self):
     def populate_craigslist_data(self):
         # Clone structure of Craigslist
         self.stdout.write('Loading Sections')
-        sections = json.load(open('demo/craigslist_data/Types.json'))
+        sections = json.load(open('allauthdemo/craigslist_data/Types.json'))
```

Comment:
Can we keep the original `demo` folder? Having `allauthdemo` folder looks more like we have a demo project for the `allauth` app, instead of the classified.

Reply:
Removed references to ```allauthdemo```, now just ```demo```.